recipes/taskgraph/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "taskgraph" %}
+{% set version = "0.9.1" %}
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: d94f63039711607a059465f76a93b9c2dc4d68105978e1e1501a9a6dd1119043
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Can this be `noarch: python`?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Yep!  Patched in 0ce72f129